central/main.go

```diff
@@ -435,6 +436,7 @@ func servicesToRegister() []pkgGRPC.APIService {
 		collectionService.Singleton(),
 		policyCategoryService.Singleton(),
 		processListeningOnPorts.Singleton(),
+		vulnMgmtService.Singleton(),
```

Comment:
Although it is not directly related to the change at hand, I think it would make sense to have this list alphabetically ordered.

Reply:
I can make a follow up for this.

Reply:
follow up: https://github.com/stackrox/stackrox/pull/11658